tsec-http4s/src/main/scala/tsec/authentication/package.scala

```diff
@@ -114,6 +131,62 @@ package object authentication {
       Kleisli.liftF(OptionT.none)
   }
 
+  type UserAwareService[Ident, A, F[_]] =
+    Kleisli[OptionT[F, ?], UserAwareRequest[F, Ident, A], Response[F]]
+
+  type UserAwareMiddleware[F[_], Ident, A] =
```

Comment:
`Ident` == `I` ?

Reply:
yeah, same thing. 

`Ident` is any `Identity`

Reply:
for consistency we should change to `I`, what do you think ?

Reply:
I don't disagree, sure. but `Ident` is also on `TSecAuthService`. Would you mind pushing to my branch real quick? I'm off to bed :)

Reply:
:+1: